cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -187,8 +197,23 @@ generateNewEpochEvents env details = do
         , esEpochNo = Strict.Just currentEpochNo
         }
 
+getTopLevelConfig :: SyncEnv -> TopLevelConfig CardanoBlock
```

Comment:
I'm unsure about this one as it's used in places that seem like it should only work if there is a Ledger